packages/razzle/config/env.js

```diff
@@ -74,7 +74,7 @@ function getClientEnvironment(target, options) {
         PUBLIC_PATH: process.env.PUBLIC_PATH || '/',
         // CLIENT_PUBLIC_PATH is a PUBLIC_PATH for NODE_ENV === 'development' && BUILD_TARGET === 'client'
         // It's useful if you're running razzle in a non-localhost container. Ends in a /
-        CLIENT_PUBLIC_PATH: process.env.CLIENT_PUBLIC_PATH || '/',
+        CLIENT_PUBLIC_PATH: process.env.CLIENT_PUBLIC_PATH,
```

Comment:
This was broken since #761

Reply:
https://github.com/jaredpalmer/razzle/pull/779